tools/prepare-release.sh

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env bash
+# -*- shell-script[bash] -*-
```

Comment:
What s the purpose of this?

Reply:
that's a "Hey Emacs" line. I think it's not needed as we have a she-bang. I can remove those.